assets/src/edit-story/components/helpCenter/test/useHelpCenter.js

```diff
@@ -113,22 +120,35 @@ describe('deriveState', () => {
 });
 
 describe('useHelpCenter', () => {
-  it('always returns the same actions by reference', () => {
+  beforeEach(() => {
+    useAPI.mockReturnValue({
+      getCurrentUser: jest.fn(() => ({ meta: {} })),
+      updateCurrentUser: jest.fn(),
+    });
+  });
+
```

Comment:
Instead of mocking the `useAPI` file it's better to provide the actual context when rendering the hook. Example:


```js
const apiContextValue = {
  actions: {
    getCurrentUser: jest.fn(() => ({ meta: {} })),
    updateCurrentUser: jest.fn(),
  },
};

// eslint-disable-next-line react/display-name, react/prop-types
const wrapper = ({children}) => (
  <ConfigContext.Provider value={configValue}>
    <APIContext.Provider value={apiContextValue}>
      {params.children}
    </APIContext.Provider>
  </ConfigContext.Provider>
);

const { result } = renderHook(() => useHelpCenter(), { wrapper });
```

To make it more DRY, move this into a helper function:


```js
function arrange() {
	  const apiContextValue = {
	    actions: {
	      getCurrentUser: jest.fn(() => ({ meta: {} })),
	      updateCurrentUser: jest.fn(),
	    },
	  };
	  
	  // eslint-disable-next-line react/display-name, react/prop-types
	  const wrapper = ({children}) => (
	    <ConfigContext.Provider value={configValue}>
	      <APIContext.Provider value={apiContextValue}>
	        {params.children}
	      </APIContext.Provider>
	    </ConfigContext.Provider>
	  );
	  
	  return renderHook(() => useHelpCenter(), { wrapper });
}
```

If you need to pass any initial data, add these as args to the function.

There's plenty of prior art in the code base.